python_ta/reporters/node_printers.py

```diff
@@ -36,12 +36,29 @@ def render_generic(msg, node=None, source_lines=None, config=None):
                 source_lines[start_line - 1],
             )
         else:
-            yield (start_line, slice(start_col, None), LineType.ERROR, source_lines[start_line - 1])
-            yield from (
-                (line, slice(None, None), LineType.ERROR, source_lines[line - 1])
-                for line in range(start_line + 1, end_line)
-            )
-            yield (end_line, slice(None, end_col), LineType.ERROR, source_lines[end_line - 1])
+            if end_line - start_line <= MAX_SNIPPET_LINES:
```

Comment:
Follow the same implementation currently in use in `render_trailing_newlines` and `render_pep8_errors_e303_and_e304`